fast-logger/fast-logger.cabal

```diff
@@ -48,6 +49,7 @@ Test-Suite spec
 
     Ghc-Options:     -Wall -threaded
     Other-Modules:   FastLoggerSpec
+    build-tool-depends: hspec-discover:hspec-discover
```

Comment:
This field breaks the backword compatibility of old Cabals.
Old cabal-install abnormally exits if this field exists.